### PR TITLE
[wip] [jsk_maps] Add change-floor server to change map

### PR DESCRIPTION
**Pull request by iory**

### CODE REVIEW

File: jsk_maps/src/change-floor-server.l [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      (format nil "floor changed to ~A" target-floor-name))
+    (change-floor target-floor-name
+                  :topicname *topicname*
+                  :base-frame-name *base-frame-name*
+                  :map-tf-mux *map-tf-mux*)
+    ;; change publish_freuquency to reload costmap automatically
+    (let ((current-costmap-publish-frequency
+           (ros::get-param (format nil "~A/~A/publish_frequency"
+                                   *move-base-node* *costmap-name*))))
+      (unix::usleep (* 1000 1000))
```
Comment (YutoUchimi) [inline jsk_maps/src/change-floor-server.l]: `(unix:sleep 1)`?

File: jsk_maps/src/change-floor.l [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env roseus
```
Comment (YutoUchimi) [inline jsk_maps/src/change-floor.l]: This  file seems to be a module. Need to be executable?

File: jsk_maps/src/change-floor.l [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          (list (instance dynamic_reconfigure::DoubleParameter :init
+                          :name "publish_frequency" :value frequency)))
+    (ros::service-call (format nil "~A/~A/set_parameters" node-name costmap-name) req))
+  t)
+
+
+(defun change-floor (target-floor
+                     &key
+                     (floor-frame "/map")
+                     (topicname "/inamenitialpose3d")
```
Comment (YutoUchimi) [inline jsk_maps/src/change-floor.l]: `(topicname "/initialpose3d")`?

**Comment by k-okada** [comment]
why you need this node ? does current demo code can not change the floor ?

**Comment by iory** [comment]
work in progress. please wait a moment for demo

**Comment by k-okada** [comment]
@iory can you fix pr2 robot to use `change-fllor-server.l` ? I am afraid we have to maintain two code for same functionality https://github.com/jsk-ros-pkg/jsk_demos/pull/1359

**Comment by iory** [comment]
OK. I'll change it to use `change-floor-server.l`.

**Pull request merged**